src/actions/connections.js

```diff
@@ -0,0 +1,29 @@
+import { LODAD_CONNECTIONS } from '../constants/action-types';
+
+
+export function loadConnections() {
+  const connections = [
+    {
```

Comment:
perhaps your stub can already follow the one proposed: https://github.com/sqlectron/sqlectron/wiki/Database-connections-file-descriptor